### PR TITLE
Updating podtato-head predefined workflow

### PR DESCRIPTION
**Pull request by oumkale**
Signed-off-by: Oum Kale <oumkale@chaosnative.com>

This PR is adding new experiment pod-dns-scoof for podtato-head predefined workflow.
 Where spoofing from an ```podtato-hats-new``` source as being from a ```podtato-hats```

### CODE REVIEW

**Comment by oumkale** [comment]
Application before spoofing :+1: 

Screenshot from 2021-05-17 12-22-19

Application after spoofing :+1: 

Screenshot from 2021-05-17 13-24-42

File: workflows/podtato-head/workflow.yaml [hunk 1]
```diff
@@ -99,6 +101,46 @@ spec:
...
+                  auxiliaryAppInfo: ""
+                  chaosServiceAccount: litmus-admin
+                  jobCleanUpPolicy: retain
+                  experiments:
+                    - name: pod-dns-spoof
+                      spec:
+                        components:
+                          env:
+                            - name: SPOOF_MAP
+                              value: '{"podtato-hats.{{workflow.parameters.adminModeNamespace}}.svc.cluster.local":"podtato-hats-new.litmus.svc.cluster.local"}'
```
Comment (ispeakc0de) [inline workflows/podtato-head/workflow.yaml]:
```suggestion
                              value: '{"podtato-hats.{{workflow.parameters.adminModeNamespace}}.svc.cluster.local":"podtato-hats-new.{{workflow.parameters.adminModeNamespace}}.svc.cluster.local"}'
```

File: workflows/podtato-head/workflow_cron.yaml [hunk 2]
```diff
@@ -103,6 +105,46 @@ spec:
...
+                  auxiliaryAppInfo: ""
+                  chaosServiceAccount: litmus-admin
+                  jobCleanUpPolicy: retain
+                  experiments:
+                    - name: pod-dns-spoof
+                      spec:
+                        components:
+                          env:
+                            - name: SPOOF_MAP
+                              value: '{"podtato-hats.{{workflow.parameters.adminModeNamespace}}.svc.cluster.local":"podtato-hats-new.litmus.svc.cluster.local"}'
```
Comment (ispeakc0de) [inline workflows/podtato-head/workflow_cron.yaml]:
```suggestion
                              value: '{"podtato-hats.{{workflow.parameters.adminModeNamespace}}.svc.cluster.local":"podtato-hats-new.{{workflow.parameters.adminModeNamespace}}.svc.cluster.local"}'
```

**Comment by ksatchit** [comment]
Great! This needs a PR into 1.13.x branch of chaos-charts as well (I see one for master: https://github.com/litmuschaos/chaos-charts/pull/458?

**Comment by oumkale** [comment]
Once this PR got merged will raise another PR for that. I need to re-fork repo again since it is a old fork 1.13.x branch is not there.

**Review by ispeakc0de — DISMISSED** [review]

**Review by ksatchit — APPROVED** [review]

**Pull request merged**